src/components/approve/CustomFeeAlerts.blocks.tsx

```diff
@@ -27,27 +42,35 @@ export const HigherFeeWarning = ({ averageFee, coin }: { averageFee: number; coi
     );
 };
 
-export const HigherFeeBanner = ({
+export const FeeBanner = ({
     averageFee,
     coin,
     onClose,
+    customFeeState,
 }: {
     averageFee: number;
     coin: string;
     onClose: () => void;
+    customFeeState: string | null;
 }) => {
+    const { t } = useTranslation();
     const formattedAmount = cropToMaxDigits({
         value: averageFee,
-        maxDigits: 3,
+        maxDigits: 4,
     });
 
     return (
         <div className='absolute flex w-full flex-row items-center justify-between gap-4 border-b border-theme-warning-500 bg-theme-warning-50 px-4 py-2 text-theme-warning-600 dark:border-theme-warning-400 dark:bg-[#4B4133] dark:text-theme-warning-400'>
             <div className='flex items-center gap-2'>
                 <Icon icon='information-circle' className='h-5 w-5 flex-shrink-0' />
                 <span className='text-sm leading-[17.5px]'>
-                    The fee specified appears to be higher than the typical rate, which is{' '}
-                    {formattedAmount} {coin}.
+                    {t('MISC.FEE_WARNING.TOP_LINE', {
+                        state: customFeeState,
+                    })}{' '}
+                    {t('MISC.FEE_WARNING.BOTTOM_LINE', {
+                        formattedAmount,
+                        coin,
+                    })}
```

Comment:
did this not work on a single line without splitting it up in translation files? 🤔